### PR TITLE
feat(commerce): args fixes #489

### LINKED ISSUE #489: All commerce commands should be able to accept appropriate arguments (defined by devdocs)
**Is your feature request related to a problem? Please describe.**
* All commerce commands should be able to accept appropriate arguments (defined by devdocs)
* Our first iteration of this will be require that the environmentId is passed explicitly each time a command is run. Future implementation will make this flexible for the user. 

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by rwalter215**
All relevant commerce commands can now handle the optional args and flags as defined by devdocs: https://devdocs.magento.com/guides/v2.4/reference/cli/magento.html#maintenanceenable

## Description

* cache:clean, cache:flush, and reindex can now handle their optional args
* all commerce commands can pass common commerce flags
* updated unit tests

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by rwalter215** [comment]
This PR contains work included in PR #488 please do not review this PR until that PR is merged.

**Comment by rwalter215** [comment]
Due to some last minute deadline chaos, this PR now also has code for issue #493. I hope this doesn't cause any confusion or breaches of best-practise. If so, let me know how to rectify.

File: src/common-commerce-flags.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+const { flags } = require('@oclif/command')
+
+module.exports = {
+  global: {
```
Comment (justinedelson) [inline src/common-commerce-flags.js:15]: This appears to be unnecessarily duplicated from `common-flags.js`

File: src/cloudmanager-helpers.js [hunk 2]
```diff
@@ -296,6 +296,16 @@ function handleError (_error, errorFn) {
   })
 }
 
+function getFormattedFlags (flags) {
+  return Object.keys(flags).filter(flag => flag !== 'programId').map(flag => {
```
Comment (justinedelson) [inline src/cloudmanager-helpers.js:300]: This is a bit hardcody and is already missing the `imsContextName` flag. See https://github.com/rwalter215/aio-cli-plugin-cloudmanager/pull/1 for an alternate approach which is more dynamic.

Comment (rwalter215) [reply src/cloudmanager-helpers.js]: Your solution looked great. Thanks for taking the time to flesh that out. I merged your changes into my branch and this PR should now be ready to merge

**Comment by justinedelson** [comment]
Merged. Let's see is semantic-release actually handles fixing two issues with one commit (which yes, is not really "best practice" but they're closely related)

**Pull request merged**